### ISSUE TITLE
Querying with '!' or 'not' converts string to regex and uses $ne

### ISSUE THREAD

**Issue by deesej** [balderdashy/sails-mongo#196]
When running a query such as
>Order.find({status: {'!': 'shipped'}}).exec(console.log);

the string 'shipped' gets converted to regex /^shipped$/i and uses $ne to query mongo. This causes mongo tho throw:
>Can't canonicalize query: BadValue Can't have regex as arg to $ne. ns:dev.order query:{ status: { $ne: /^shipped$/i } }

**Comment by sailsbot** [comment]
Thanks for posting, @deesej.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
